praw/const.py

```diff
@@ -2,7 +2,7 @@
 import sys
 
 
-__version__ = '6.1.0'
+__version__ = '6.2.0.dev0'
```

Comment:
Until we have an actual addition, can we make this `6.1.1.dev0`? The reason being is in the event we need to fix something broken we don't end up in a case where we need to release an older version that what someone might have installed.

Reply:
Yep!